src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,169 @@
+package duke;
+
+import exceptions.DukeException;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static Task[] taskList = new Task [100];
+    static int taskIndex=0;
+
+    public static void main(String[] args) {
+
+        greetUser("hi");
+        startChatbot();
+        greetUser("bye");
+    }
+
+    private static void startChatbot() {
+        Scanner input = new Scanner(System.in);
+        String task = "";
+        while(true) {
+            try {
+                task = input.nextLine();
+                System.out.println("    ____________________________________");
```

Comment:
Avoid magic literals. Consider declaring a new variable for the string (e.g. `MESSAGE_BORDER`).